server/utils/constants.ts

```diff
@@ -54,3 +54,5 @@ export enum AD_DOC_FIELDS {
   ERROR = 'error',
   INDICES = 'indices',
 }
+
+export const MAX_MONITORS = 1000;
```

Comment:
you may put it [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/development/public/utils/constants.ts#L46)

Reply:
Thanks. Prefer to keep this as server size constraints currently. As it's used only by alerting routes logic from service side.